data-transformers/transformers/fabanTransformer.py

```diff
@@ -0,0 +1,285 @@
+    query["metric_unit"] = data.find("benchSummary").find("metric").attrib["unit"]
+    query["metric_value"] = data.find("benchSummary").find("metric").text
+    query["passed"] = data.find("benchSummary").find("passed").text
+    return query
+
+def createDriverSummaryQuery(data, trialID, experimentID, host):
+    queries = []
+    for driver in data.findall("driverSummary"):
+        query = {}
+        query["trial_id"] = trialID
+        query["experiment_id"] = experimentID
+        query["host"] = host
+        query["name"] = driver.attrib["name"]
+        query["metric_unit"] = driver.find("metric").attrib["unit"]
+        query["metric_value"] = driver.find("metric").text
+        query["start_time"] = driver.find("startTime").text
+        query["end_time"] = driver.find("endTime").text
+        query["total_ops_unit"] = driver.find("totalOps").attrib["unit"]
+        query["total_ops_value"] = driver.find("totalOps").text
+        query["users"] = driver.find("users").text
+        query["rtxtps"] = driver.find("rtXtps").text
+        query["passed"] = driver.find("passed").text
+        query["mix_allowed_deviation"] = driver.find("mix").attrib["allowedDeviation"]
+        query["response_times_unit"] = driver.find("responseTimes").attrib["unit"]
+        queries.append(query)
+    return queries
+
+def createDriverMixQuery(data, trialID, experimentID, host):
+    queries = []
+    for driver in data.findall("driverSummary"):
+        for operation in driver.find("mix"):
+            query = {}
+            query["trial_id"] = trialID
+            query["experiment_id"] = experimentID
+            query["host"] = host
+            query["driver_name"] = driver.attrib["name"]
+            query["op_name"] = operation.attrib["name"]
+            query["allowed_deviation"] = driver.find("mix").attrib["allowedDeviation"]
+            query["successes"] = operation.find("successes").text
+            query["failures"] = operation.find("failures").text
+            query["mix"] = operation.find("mix").text
+            query["required_mix"] = operation.find("requiredMix").text
+            query["passed"] = operation.find("passed").text
+            queries.append(query)
+    return queries
+
+def createDriverResponseTimesQuery(data, trialID, experimentID, host):
+    queries = []
+    for driver in data.findall("driverSummary"):
+        for operation in driver.find("responseTimes"):
+            for stat in operation:
+                query = {}
+                query["trial_id"] = trialID
+                query["experiment_id"] = experimentID
+                query["host"] = host
+                query["driver_name"] = driver.attrib["name"]
+                query["op_name"] = operation.attrib["name"]
+                if not "passed" in stat.tag:
+                    query["stat_name"] = stat.tag
+                    query["stat_value"] = stat.text
+                else:
+                    continue
```

Comment:
I don't want to in fact, I want to skip the passed field, because it's not a numerical stat. I'll write this more clearly by putting the check at the top.